ethcore/src/engines/mod.rs

```diff
@@ -133,6 +133,18 @@ pub enum Seal {
 /// A system-calling closure. Enacts calls on a block's state from the system address.
 pub type SystemCall<'a> = FnMut(Address, Vec<u8>) -> Result<Vec<u8>, String> + 'a;
```

Comment:
Do you think it makes sense to remove this type? It seems like everywhere `SystemCall` is supported `SystemOrCodeCall` should also be? (even if it's not exposed by any parameter).

Reply:
I think we are still using `SystemCall` in several other cases, but I do agree that we may eventually want to clean up this.